[2024-10-30T18:12:09.200+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-30T18:12:09.205+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.create_table_user_review manual__2024-10-30T18:12:05.893765+00:00 [queued]>
[2024-10-30T18:12:09.207+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.create_table_user_review manual__2024-10-30T18:12:05.893765+00:00 [queued]>
[2024-10-30T18:12:09.207+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-30T18:12:09.211+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): create_table_user_review> on 2024-10-30 18:12:05.893765+00:00
[2024-10-30T18:12:09.213+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=77) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-30T18:12:09.213+0000] {standard_task_runner.py:72} INFO - Started process 80 to run task
[2024-10-30T18:12:09.214+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'create_table_user_review', 'manual__2024-10-30T18:12:05.893765+00:00', '--job-id', '56', '--raw', '--subdir', 'DAGS_FOLDER/***.py', '--cfg-path', '/tmp/tmpfeexwuhm']
[2024-10-30T18:12:09.214+0000] {standard_task_runner.py:105} INFO - Job 56: Subtask create_table_user_review
[2024-10-30T18:12:09.228+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.create_table_user_review manual__2024-10-30T18:12:05.893765+00:00 [running]> on host 6bca134bba56
[2024-10-30T18:12:09.251+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='your_name' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='create_table_user_review' AIRFLOW_CTX_EXECUTION_DATE='2024-10-30T18:12:05.893765+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-30T18:12:05.893765+00:00'
[2024-10-30T18:12:09.252+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-30T18:12:11.066+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-30T18:12:11.079+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-30T18:12:11.080+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=create_table_user_review, run_id=manual__2024-10-30T18:12:05.893765+00:00, execution_date=20241030T181205, start_date=20241030T181209, end_date=20241030T181211
[2024-10-30T18:12:11.101+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-30T18:12:11.119+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-30T18:12:11.120+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
